Terms of Reference.md

```diff
@@ -0,0 +1,140 @@
+# Terms of Reference
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Project Scope](#project-scope)
+- [Purpose of the Technical Committee](#purpose-of-the-technical-committee)
+- [Membership Composition](#membership-composition)
+- [Project Schedule](#project-schedule)
+- [Meeting Schedule](#meeting-schedule)
+- [Consensus Building](#consensus-building)
+- [Code of Conduct](#code-of-conduct)
+
+## Overview
+
+This Terms of Reference (ToR) defines the goals of the Trust Meter Technical
+Specification, how it differs or relates to other specifications regarding
+Artificial Intelligence (AI) and Trust, and the objectives of the Technical
+Committee, outlining the roles and responsibilities of those involved.
+
+## Project Scope
+
+The Trust Meter Technical Specification addresses statistical discrimination
+against data outliers and small minorities in mechanized statistical reasoning
+in AI decision tools. The Trust Meter Technical Specification is intended to
```

Comment:
"AI decision tools" is a broad term including wide range of AI systems such as rule-based decision systems and robotics etc, while machine learning is a subset of AI that uses data, statistics and algorithms to learn and make decisions. Since this spec focuses on preventing statistical discrimination, which is an issue that arises specifically from data-driven decision-making. Would it be clearer to mention the spec applies to machine learning-based decision tools?

Reply:
While I agree with you, @cindyli, this is the scope clause.  @jobara said that since the scope has already been approved by SCC, it cannot change.  Or, it is difficult to change at this point?  This text is almost identical to that used in the the [SOW](https://ocaduniversity.sharepoint.com/:w:/r/teams/IDRC/_layouts/15/Doc.aspx?sourcedoc=%7BA2B286CC-DBB3-4CD4-A153-74CB446AF60A%7D&file=National-TS-SOW_TrustMeter.docx&action=default&mobileredirect=true) that SCC accepted.

Maybe this can be handled in the "Purpose of the Technical Committee" section as you noted in another of your [comments](https://github.com/inclusive-design/trust-meter/pull/57#discussion_r2072101994).